controllers/nnf_workflow_controller.go

```diff
@@ -344,7 +382,7 @@ func (r *NnfWorkflowReconciler) finishProposalState(ctx context.Context, workflo
 
 	// Wait for the breakdown to be ready
 	if directiveBreakdown.Status.Ready != ConditionTrue {
-		return &ctrl.Result{}, nil
+		return Requeue.withObject(directiveBreakdown).withReason("status pending"), nil
```

Comment:
Could you do something like result.Requeue("status pending") to make this easier?